fccodegen/data/tests/weblab_model.pyx

```diff
@@ -72,36 +82,92 @@ cdef int _EvaluateRhs(Sundials.realtype var_environment__time, Sundials.N_Vector
     (<Sundials.N_VectorContent_Serial>ydot.content).data[3] = d_dt_n
 
 
-cdef class GeneratedModel_Proto_tmpHzG5bQ(CvodeSolver):
+cdef class GeneratedModel_Proto_tmpHzG5bQ(CvodeSolver): # TODO Generate name
+
+    # The name of the free variable, for use in the ModelWrapperEnvironment
+    # From: fc.simulations.AbstractOdeModel
     cdef public char* freeVariableName
+
+    # The value of the free variable
+    # From: fc.simulations.AbstractOdeModel
     cdef public double freeVariable
+
+    # A mapping from variable name to index within the state variable vector,
+    # for use in the ModelWrapperEnvironment
+    # From: fc.simulations.AbstractOdeModel
     cdef public object stateVarMap
+
+    # A numpy array containing initial values for the state variables
+    # From: fc.simulations.AbstractOdeModel
     cdef public np.ndarray initialState
+
+    # A mapping from parameter name to index within the parameters vector, for
+    # use in the ModelWrapperEnvironment
+    # From: fc.simulations.AbstractOdeModel
     cdef public object parameterMap
+
+    # A numpy array containing model parameter values
+    # From: fc.simulations.AbstractOdeModel
     cdef public np.ndarray parameters
+
+    # An ordered list of the names of the model outputs, as they will be
+    # returned by GetOutputs
+    # From: fc.simulations.AbstractOdeModel
     cdef public object outputNames
 
+    # Mapping from names to saved model states.
+    # From: fc.simulations.AbstractOdeModel
     cdef public object savedStates
+
+    # An instance of fc.utility.environment.ModelWrapperEnvironment
+    # TODO: What's this?
+    # From: fc.simulations.AbstractOdeModel
     cdef public object env
+
+    # True if the solver needs to be reset due to a model change.
+    # TODO This is only ever set via the "environment". Not sure why.
+    # From: fc.simulations.AbstractOdeModel
     cdef public bint dirty
+
+    # TODO Not documented in AbstractModel
+    # From: fc.simulations.AbstractModel
     cdef public char* outputPath
+
+    # Level of indentation to use for progress output.
+    # From: fc.simulations.AbstractModel
     cdef public object indentLevel
 
+    # Link to generated module.
+    # Set in: fc.utility.protocol.Protocol
+    # TODO: Nobody seems to ever access this variable. Is it just to prevent
+    # garbage collection?
     cdef public object _module
+
+    # TODO What's this?
```

Comment:
It's the environment for the simulation running this model, which is mainly useful when evaluating `set variable` type modifiers during the course of a simulation.